<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsGrassOptionsBase</class>
 <widget class="QDialog" name="QgsGrassOptionsBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GRASS Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QSplitter" name="mOptionsSplitter">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QFrame" name="mOptionsListFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QListWidget" name="mOptionsListWidget">
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideNone</enum>
         </property>
         <property name="resizeMode">
          <enum>QListView::Adjust</enum>
         </property>
         <property name="uniformItemSizes">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>General</string>
          </property>
          <property name="icon">
           <iconset resource="../../../images/images.qrc">
            <normaloff>:/images/themes/default/propertyicons/general.svg</normaloff>:/images/themes/default/propertyicons/general.svg</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Modules</string>
          </property>
          <property name="icon">
           <iconset resource="../../../images/images.qrc">
            <normaloff>:/images/themes/default/grass_mapset.png</normaloff>:/images/themes/default/grass_mapset.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Browser</string>
          </property>
          <property name="toolTip">
           <string>Browser</string>
          </property>
          <property name="icon">
           <iconset resource="../../../images/images.qrc">
            <normaloff>:/images/icons/qbrowser-icon-60x60.png</normaloff>:/images/icons/qbrowser-icon-60x60.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Region</string>
          </property>
          <property name="icon">
           <iconset resource="../../plugins/grass/qgsgrass_plugin.qrc">
            <normaloff>:/images/themes/default/grass/grass_options_region.png</normaloff>:/images/themes/default/grass/grass_options_region.png</iconset>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="mOptionsFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QStackedWidget" name="mOptionsStackedWidget">
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="mGeneralPage">
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QLabel" name="mGrassVersionLabel">
             <property name="toolTip">
              <string>The version of GRASS which was used to build the GRASS provider and plugin in QGIS. Exactly the same version must be used on runtime.</string>
             </property>
             <property name="text">
              <string notr="true">GRASS version</string>
             </property>
             <property name="margin">
              <number>3</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QgsCollapsibleGroupBox" name="mGisbaseGroupBox">
             <property name="title">
              <string>GRASS installation</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <item>
               <widget class="QRadioButton" name="mGisbaseDefaultRadioButton">
                <property name="text">
                 <string>Default</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QRadioButton" name="mGisbaseCustomRadioButton">
                  <property name="text">
                   <string>Custom</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="mGisbaseLineEdit"/>
                </item>
                <item>
                 <widget class="QPushButton" name="mGisbaseBrowseButton">
                  <property name="text">
                   <string>Browse</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="mGisbaseErrorLabel">
                <property name="text">
                 <string>GIsbase error</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>165</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="mModulesPage">
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QScrollArea" name="mModulesScrollArea">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="mModulesContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>247</width>
                <height>141</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QgsCollapsibleGroupBox" name="mModulesConfigGroupBox">
                 <property name="title">
                  <string>Modules interface configuration</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <widget class="QRadioButton" name="mModulesConfigDefaultRadioButton">
                    <property name="text">
                     <string>Default</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                     <widget class="QRadioButton" name="mModulesConfigCustomRadioButton">
                      <property name="text">
                       <string>Custom</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="mModulesConfigDirLineEdit"/>
                    </item>
                    <item>
                     <widget class="QPushButton" name="mModulesConfigBrowseButton">
                      <property name="text">
                       <string>Browse</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="mModulesDebugCheckBox">
                 <property name="text">
                  <string>Debug mode</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>246</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="mBrowserPage">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QScrollArea" name="mBrowserScrollArea">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="mBrowserContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>426</width>
                <height>329</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_13">
               <item>
                <widget class="QgsCollapsibleGroupBox" name="mImportGroupBox">
                 <property name="title">
                  <string>Import</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                     <widget class="QLabel" name="mCrsTransformationLabel">
                      <property name="text">
                       <string>CRS transformation</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QComboBox" name="mCrsTransformationComboBox">
                      <property name="toolTip">
                       <string>Approximate CRS transformation is fast but it may be inaccurate.</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="mImportExternalCheckBox">
                    <property name="toolTip">
                     <string>Create a link to the external data for GDAL data sources with the same CRS as target mapset by r.external, instead of making copy of data.</string>
                    </property>
                    <property name="text">
                     <string>Create link to external data if possible</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QgsCollapsibleGroupBox" name="mLayersBox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="title">
                  <string>Layers</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_12">
                  <item>
                   <widget class="QCheckBox" name="mTopoLayersCheckBox">
                    <property name="text">
                     <string>Show virtual topological layers</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="mRegionPage">
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QgsCollapsibleGroupBox" name="mRegionGroupBox">
             <property name="title">
              <string>Region border</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <widget class="QLabel" name="mRegionColorLabel">
                  <property name="text">
                   <string>Color</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QgsColorButtonV2" name="mRegionColorButton">
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>120</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="mRegionWidthLabel">
                  <property name="text">
                   <string>Width</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QDoubleSpinBox" name="mRegionWidthSpinBox">
                  <property name="decimals">
                   <number>1</number>
                  </property>
                  <property name="singleStep">
                   <double>0.200000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="mButtonBoxFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsColorButtonV2</class>
   <extends>QToolButton</extends>
   <header>qgscolorbuttonv2.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../images/images.qrc"/>
  <include location="../../plugins/grass/qgsgrass_plugin.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>mOptionsListWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>mOptionsStackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
